centreon/src/Core/Contact/Infrastructure/Repository/DbReadContactGroupRepository.php

```diff
@@ -56,6 +56,49 @@ public function __construct(DatabaseConnection $db, SqlRequestParametersTranslat
         ]);
     }
 
+    /**
+     * @inheritDoc
+     */
+    public function findNamesByIds(int ...$ids): array
+    {
+        if ([] === $ids) {
+            return [];
+        }
+
+        $ids = array_unique($ids);
+
+        $fields = '';
+        foreach ($ids as $index => $id) {
+            $fields .= ('' === $fields ? '' : ', ') . ':id_' . $index;
```

Comment:
```suggestion
            $fields .= ('' === $fields ?: ', ') . ':id_' . $index;
```
As the true operation is the same that your initial value you can use the elvis operator, so if the condition is true, nothing happened, if its false you gonna concatenate ', '

Reply:
btw nice way to handle those dynamic token instead of creating a new array and looping on array keys 👍

Reply:
Can be replace by using the trait SqlMultipleBindTrait::createMultipleBindQuery

Reply:
Yep can be don't mind if I do this in the following PR ? It is noted 👍🏻